yapapi/rest/payment.py

```diff
@@ -67,13 +70,15 @@ class Allocation(_Link):
     expires: Optional[datetime]
     "Allocation expiration timestamp"
 
+    @repeat_on_error(max_tries=5)
     async def details(self) -> AllocationDetails:
```

Comment:
I think that this method is never called, so there is no need to add `@repeat_on_error` here.